config/trackers.go

```diff
@@ -13,3 +13,8 @@ var Trackers = []string{
 	"udp://tracker.internetwarriors.net:1337/announce",
 	"http://mgtracker.org:6969/announce",
 	"http://tracker.baka-sub.cf/announce"}
+
+// NeededTrackers : Array indexes of Trackers for needed tracker in a torrent file
```

Comment:
Dunno, that's our tracker and it was decided to implement it in our torrents?
I think there was an issue and then they added the information on upload page